Sources/Commands/SwiftTool.swift

```diff
@@ -366,6 +366,11 @@ public class SwiftTool<Options: ToolOptions> {
             option: parser.add(option: "--trace-resolver", kind: Bool.self),
             to: { $0.enableResolverTrace = $1 })
 
+        binder.bind(
+            option: parser.add(option: "--numThreads", shortName: "-j", kind: Int.self,
```

Comment:
How about `--num-jobs` instead of `numThreads`?

Reply:
It's worth noting that [`swiftc` distinguishes `-num-threads` and `-j`](https://github.com/apple/swift/blob/master/docs/Driver.md#whole-module-optimization) for a (subtle) reason:

> Threaded mode is controlled by the `-num-threads` command-line option rather than `-j` used to control the number of jobs (simultaneous subprocesses spawned by the driver). Why? While changing the number of jobs should never affect the final product, using threaded vs. non-threaded compilation does.

I don't think SwiftPM needs to have two settings for this, but I'd prefer if it not mix the names.

Reply:
This actually getting passed to llbuild, not swiftc. llbuild calls this `--jobs`/`-j`. Why don't I match llbuild?